tools/labs/qemu/create_net.sh

```diff
@@ -11,10 +11,10 @@ device=$1
 USER=$(whoami)
 
 case "$device" in
-    "tap0")
+    "lkt-tap0")
```

Comment:
The usage on line 5 needs to be updated accordingly.

Reply:
Fixed, thanks!